src/Pages/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import './styledComponent.js';
+import { getContainerSize } from '../Constants/getDimension';
+import {debounce} from '../Constants/debounce';
+import { ParentResizableBox } from './styledComponent';
+
+function ResizableBox() {
+    const [dimension, setDimension] = useState({
+        screenWidth: "",
+        screenHeight: ""
+    })
+    useEffect(() => {
+        const result = getContainerSize()
+        setDimension({ ...dimension, screenHeight: result.screenHeight, screenWidth: result.screenWidth })
+    }, []);
+
+    const panel = document.getElementById("resizavle_div");
+    let m_pos;
+    function resize(e) {
+        const dx = m_pos - e.x;
+        m_pos = e.x;
```

Comment:
why are we calculating position etc. we just need width height of screen. posotion of mouse kahan se agaya